ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java

```diff
@@ -204,6 +204,85 @@ public final class RespConstantsV7 {
     public static final String TECKEN_SVIKTANDE_KOGNITIV_FUNKTION_JSON_ID = "sviktandeKognitivFormaga";
     public static final String TECKEN_SVIKTANDE_KOGNITIV_FUNKTION_SVAR_TEXT_ID = "FRG_23.RBK";
 
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_CATEGORY_ID = "somnellervakenhetsstorning";
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_CATEGORY_TEXT_ID = "KAT_10.RBK";
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_SVAR_ID_24 = "24";
```

Comment:
Should we remove the trailing number from the constant to keep the format same for all?

Reply:
Will fix!